src/Microsoft.AspNet.Diagnostics/Views/BaseView.cs

```diff
@@ -117,6 +118,48 @@ protected void WriteLiteral(object value)
                 values);
         }
 
+        private List<string> _attributeValues { get; set; }
```

Comment:
I was under the impression that private members were [_camelCase](https://github.com/aspnet/Home/wiki/Engineering-guidelines#coding-style-guidelines--general)? Or is that only on fields and not properties?